### DISCUSSION TITLE
Where are the releases?

### DISCUSSION THREAD

**Discussion by ilonachan** [SceneGate/SceneGate#39] [General]
I want to use this program. I'm not an expert, have not looked into how the tool works internally, and wasn't planning to, I just want to extract an nds game. The readme says this should be as simple as unzipping and running the shipped portable ZIP file... what ZIP file though? There are no releases, and the github.io page doesn't have any files either. Am I blind and missing something obvious??

By the way, I've then tried to compile the program from source using the instructions in the readme, and those also don't work. I get the following error:
```
/mnt/general/games/nds/tools/SceneGate/build/orchestrator/BuildSystem.csproj : error NU1101: Unable to find package Microsoft.NETCore.App.Host.arch-x64. No packages exist with this id in source(s): nuget.org. PackageSourceMapping is enabled, the following source(s) were not considered: PleOps-Preview. [/mnt/general/games/nds/tools/SceneGate/build/orchestrator/BuildSystem.sln]
```
I've heard of an error like this (https://github.com/dotnet/sdk/issues/40676) related to transition to .NET 8.0, but I can't do anything about this because I don't know how .NET tooling and development works! I expected this app to work as expected, or at least *run* out of the box!

**Comment by pleonex** [comment]
Thanks for the interest! There aren't releases as the program is still in an early development phase. There are frequent crashes (especially with the hex viewer) and missing features required to continue supporting formats. I wanted to do a last refactor before making a beta release but then life happened.

~~In the meantime, I just re-triggered the build pipeline so you can download the artifacts [here](https://github.com/SceneGate/SceneGate/actions/runs/10591663114). Unfortunately, it will expire in about a week. I'll try to upload them as a beta release in GitHub later today.~~

Regarding the build error, I just tried from Ubuntu in WSL and it works for me. Can you give me more details of your environment? The output from the following commands may help to try to reproduce on my side:

- OS: `cat /etc/*release`
- Is the OS physical / Windows WSL / virtual machine?
- .NET SDK: `dotnet --list-sdks`
- .NET runtime: `donet --list-runtimes`
